src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -65,6 +66,21 @@ export const CloudEnvironmentModal = ({
     }
   })
 
+  const renderAzureModal = () => h(AzureComputeModalBase, {
+    isOpen: viewMode === NEW_AZURE_MODE,
+    shouldHideCloseButton: true,
```

Comment:
Nit: consider making `shouldHideCloseButton` true by default since you will never want to show it.

Reply:
it needs to be shown when its open from the `+Start` button, but not from the cloud env modal

Reply:
Ah, interesting! I never realized until now that the 2 paths are at all different (I thought the Start button was just a shortcut to getting to the settings page for a tool).